projects/Mallard/ophan/build.gradle

```diff
@@ -61,6 +88,21 @@ task copyFramework {
     }
 }
 
+task copyFatFramework {
+    dependsOn tasks.fatFramework
+
+    doLast {
+        def srcFile = tasks.fatFramework.outputs.files
+        def targetDir = getProperty('configuration.build.dir')
+        println("srcFile=$srcFile")
```

Comment:
Do you think these` printlns` will be useful to keep in?

Reply:
No, I'll get rid of them